### ISSUE TITLE
设置0.0标志位时会报错， The shape of the mask [2006] at index 0 does not match the shape of the indexed tensor [512, 4] at index 0

### ISSUE THREAD

**Issue by 2807402145** [hding2455/DSC#4]
bbox_pred_for_regress = torch.zeros_like(bbox_results['bbox_pred'])
                    bbox_pred_for_regress[:, :] = bbox_results['bbox_pred'][:, :]
                    bbox_pred_for_regress[real_gt_flags] = 0.0

**Comment by hding2455** [comment]
If there is any update, please let me know.
